test/OpenTelemetry.Tests/Metrics/AggregatorTestsBase.cs

```diff
@@ -239,47 +239,53 @@ public void MultiThreadedHistogramUpdateAndSnapShotTest()
     }
 
     [Theory]
-    [InlineData("Microsoft.AspNetCore.Hosting", "http.server.request.duration")]
-    [InlineData("Microsoft.AspNetCore.Http.Connections", "signalr.server.connection.duration")]
-    [InlineData("Microsoft.AspNetCore.RateLimiting", "aspnetcore.rate_limiting.request_lease.duration")]
-    [InlineData("Microsoft.AspNetCore.RateLimiting", "aspnetcore.rate_limiting.request.time_in_queue")]
-    [InlineData("Microsoft.AspNetCore.Server.Kestrel", "kestrel.connection.duration")]
-    [InlineData("Microsoft.AspNetCore.Server.Kestrel", "kestrel.tls_handshake.duration")]
-    [InlineData("OpenTelemetry.Instrumentation.AspNetCore", "http.server.duration")]
-    [InlineData("OpenTelemetry.Instrumentation.Http", "http.client.duration")]
-    [InlineData("System.Net.Http", "http.client.connection.duration")]
-    [InlineData("System.Net.Http", "http.client.request.duration")]
-    [InlineData("System.Net.Http", "http.client.request.time_in_queue")]
-    [InlineData("System.Net.NameResolution", "dns.lookups.duration")]
-    [InlineData("General.App", "simple.alternative.counter")]
-    public void HistogramBucketsDefaultUpdatesForSecondsTest(string meterName, string instrumentName)
+    [InlineData("Microsoft.AspNetCore.Hosting", "http.server.request.duration", "s", KnownHistogramBuckets.DefaultShortSeconds)]
+    [InlineData("Microsoft.AspNetCore.Hosting", "http.server.request.duration", "ms", KnownHistogramBuckets.Default)]
+    [InlineData("Microsoft.AspNetCore.Hosting", "http.server.request.duration", "By", KnownHistogramBuckets.Default)]
+    [InlineData("Microsoft.AspNetCore.Hosting", "http.server.request.duration", null, KnownHistogramBuckets.Default)]
+    [InlineData("Microsoft.AspNetCore.Http.Connections", "signalr.server.connection.duration", "s", KnownHistogramBuckets.DefaultLongSeconds)]
+    [InlineData("Microsoft.AspNetCore.RateLimiting", "aspnetcore.rate_limiting.request_lease.duration", "s", KnownHistogramBuckets.DefaultShortSeconds)]
+    [InlineData("Microsoft.AspNetCore.RateLimiting", "aspnetcore.rate_limiting.request.time_in_queue", "s", KnownHistogramBuckets.DefaultShortSeconds)]
+    [InlineData("Microsoft.AspNetCore.Server.Kestrel", "kestrel.connection.duration", "s", KnownHistogramBuckets.DefaultLongSeconds)]
+    [InlineData("Microsoft.AspNetCore.Server.Kestrel", "kestrel.tls_handshake.duration", "s", KnownHistogramBuckets.DefaultShortSeconds)]
+    [InlineData("OpenTelemetry.Instrumentation.AspNetCore", "http.server.duration", "s", KnownHistogramBuckets.Default)]
+    [InlineData("OpenTelemetry.Instrumentation.Http", "http.client.duration", "s", KnownHistogramBuckets.Default)]
```

Comment:
Same as https://github.com/open-telemetry/opentelemetry-dotnet/pull/5008/files?diff=split&w=1#r1380873808

https://github.com/open-telemetry/opentelemetry-dotnet/blob/c8f939e60b4f010e8bdd54c639109adcfbcf5229/src/OpenTelemetry.Instrumentation.Http/Implementation/HttpHandlerMetricsDiagnosticListener.cs#L40
```suggestion
    [InlineData("OpenTelemetry.Instrumentation.Http", "http.client.duration", "ms", KnownHistogramBuckets.Default)]
```